common/lib/container-definitions/src/loader.ts

```diff
@@ -252,20 +252,27 @@ export interface IContainer extends IEventProvider<IContainerEvents>, IFluidRout
 
     /**
      * Boolean indicating whether the container is currently connected or not
+     * @deprecated - 0.58, This API will be removed in 0.60
+     * Use `connectionState` instead
+     * See https://github.com/microsoft/FluidFramework/issues/9167 for context
```

Comment:
To simplify it for readers, please add how connected property is calculated (i.e. what kind of change consumers need to make in their code base)